src/index.js

```diff
@@ -0,0 +1,8 @@
+function say(name){
```

Comment:
```suggestion
function say(name)
{
```
🙈 we use [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) to indent code